tuneme/templates/core/article_page.html

```diff
@@ -35,11 +35,6 @@
         <hr>
         <p class="bold">{{ block.value }}</p>
       {% else %}
-      {% if block.block_type == 'page' %}
-        <hr>
-        <div class="readmore">{% trans "You may want to read:" %}</div>
-        {{ block }}
-      {% else %}
```

Comment:
@TrevorOctober you don't need to remove the if block, you can just remove the <hr> and the `you may want to read` so you should have 
```
{% if block.block_type == 'page' %}
    {{ block }}
{% else %}
```